tests/index.js

```diff
@@ -61,6 +64,15 @@ describe("index.js", function () {
 
                     const titleRegexp = new RegExp(`^# .+ \\(${fileName}\\)$`, "m");
                     assert.ok(fileContents.match(titleRegexp), "includes rule name in title header");
+
+                    if (index.rules[fileName].meta.fixable === "code") {
```

Comment:
I think this could just check `index.rules[fileName].meta.fixable`. Another valid option is `"whitespace"`, and who knows what else may be supported by ESLint core in the future. So I'm okay with just checking against that value being truthy.

Reply:
Good point, fixed.